L1Trigger/Configuration/python/L1TRawToDigi_cff.py

```diff
@@ -33,7 +33,7 @@ def unpack_legacy():
     
 
 def unpack_stage1():
-    global csctfDigis, dttfDigis, gtDigis,caloStage1Digis,caloStage1FinalDigis
+    global csctfDigis, dttfDigis, gtDigis,caloStage1Digis,caloStage1FinalDigis,gctDigis
```

Comment:
@mulhearn - I had missed these globals.. we'll need to remove them at some point very soon. why are they needed here?